templates/tfengine/components/cicd/env/triggers/main.tf

```diff
@@ -0,0 +1,120 @@
+{{- /* Copyright 2021 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */ -}}
+locals {
+  terraform_root = var.terraform_root == "/" ? "." : var.terraform_root
+  terraform_root_prefix = local.terraform_root == "." ? "" : "${local.terraform_root}/"
+  // Github and CSR are mutually exclusive so there shouldn't be both specified on the same
+  // terraform configuration, but in case they're, priority goes to github since it creates the
+  // less amount of extra resources 
```

Comment:
Creates the less amount of extra resources don't seem to me a very convincing reason. You can just mention the priority goes to GitHub. That's good enough.

Reply:
Sg, done